src/dir.rs

```diff
@@ -460,49 +460,30 @@ impl Deref for PathDir {
     }
 }
 
-impl Into<PathAbs> for PathDir {
-    /// Downgrades the `PathDir` into a `PathAbs`
```

Comment:
Do we want to keep the docs? Probably not...

Reply:
I considered it, but all these traits have pretty solid, unambiguous definitions, and have their own documentation which we don't really need to duplicate. The more docs we have, the more docs we need to maintain, so we should concentrate on the structs and functions that are unique to this crate instead of things from the standard library.